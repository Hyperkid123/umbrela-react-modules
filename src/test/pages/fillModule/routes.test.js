import React from 'react'
import Routes from '../../../pages/fillModule/routes';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Fill module routes component', () => {
  it('should render correctly', () => {
    const tree = shallow(
      <Routes/>
    )
    expect(toJson(tree)).toMatchSnapshot();
  })
});
