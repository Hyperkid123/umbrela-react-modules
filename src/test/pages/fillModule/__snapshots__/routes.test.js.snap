// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fill module routes component should render correctly 1`] = `
<HashRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/fill/:sheetId"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/submit"
    />
  </Switch>
</HashRouter>
`;
